### ISSUE TITLE
SAMI html entities 

### ISSUE THREAD

**Issue by dseeker** [papnkukn/subsrt#4]
The video player's I've tested don't parse the text of sami files, so the output is raw html entities.

I'm not sure if this is needed. seems to me in HTML markup single and double quotes don't need to be escaped, only opening and closing tag. Is there any specification that states this is necessary?
https://github.com/papnkukn/subsrt/blob/master/lib/format/smi.js#L10

**Comment by papnkukn** [comment]
I agree, more testing should be done. Parsers are mostly done from the practical scenarios. Video players do not always follow the specifications. Anyway, examples are welcome to solve the issue.
